pom.xml

```diff
@@ -148,6 +148,7 @@
         <module>knox-homepage-ui</module>
         <module>gateway-openapi-ui</module>
         <module>knox-token-management-ui</module>
+        <module>gateway-provider-security-dos</module>
```

Comment:
Please put it under the already existing `gateway-provider-security-XXX` modules.